frontend/src/components/runs-table/runs-table.tsx

```diff
@@ -0,0 +1,146 @@
+import { useNavigate } from "react-router-dom"
+import styled from "styled-components"
```

Comment:
hier ist auch noch ein kleiner Linter Fehler

```suggestion
import { styled } from "styled-components";
```